### ISSUE TITLE
Auto Trailer Homescreen Feature?

### ISSUE THREAD

**Issue by lm209** [SerpentDrago/skin.auramod#262]
I don't know if this is actually the case, but I've tried all the settings and no trailer is played.

I Wish i could See a Trailer in the Netflix View Or does the function not exist at all?

**Comment by jojobrogess** [comment]
If you're talking about viewing the trailer from the mainscreen(like bingie/real netflix), then no.

There is an issue already up for that, and talk about whether it would be a smart idea or not to implement.
(considering api limits/processing power)
 [here:#187](https://github.com/SerpentDrago/skin.auramod/issues/187).

inb4after said this in that thread:
>The only auto-trailer options are for spotlights on homescreen and info screen as you mentioned. 

In Skin Settings> Library Views you'll see the option `trailer auto play`, but that will only activate once you've already entered a video's "information page"(see last image). 

![2022-06-15 23_52_55-Greenshot](https://user-images.githubusercontent.com/17692472/174009907-76fcc9ce-4858-455a-b160-3d35e230191e.png)

I don't use spotlights so I can't really help with setting one up correctly, if that's what you're looking for. 
Maybe someone else can help you set it up correctly?

****************************************************************************************

But if you're asking if you can play the trailer AT ALL from within the `Netflix` home style, then yes.

Select the title you wish to watch the trailer in, unless you changed it, the main selection method should take you to the `information` screen where you can click the `trailer` button.
If it doesn't, press `C` which should bring up the context menu. 
From there select `information`.

Trailer location(example) is here:
![2022-06-15 23_36_31-Greenshot](https://user-images.githubusercontent.com/17692472/174007104-bb6a2acc-899a-43af-8324-8ed3236a47c0.png)
 
****************************************************************************************

I can tell you though, I **HARD CORE** agree with this feature request. 
Although I do understand concerns.

**Comment by lm209** [comment]
yes exactly in the main menu.  too bad the function doesn't (yet) exist.  I love this skin it fits my personal ideas perfectly. clear that the performance may be affected but not impossible.  Still awesome skin.  I don't know how it works with the spotlights until now, at least I don't know exactly how it works.
